src/pandablocks_ioc/_pvi.py

```diff
@@ -60,11 +60,12 @@ def add_pvi_info_to_record(
     access: str,
 ):
     block, field = record_name.split(":", maxsplit=1)
-    block_name_suffixed = f"pvi.{field.lower().replace(':', '_')}.{access}"
+    pvi_pv = RecordName(f"{block}:PVI")
+    block_name_suffixed = f"value.{field.lower().replace(':', '_')}.{access}"
     record.add_info(
         "Q:group",
         {
-            RecordName(f"{block}:PVI"): {
+            pvi_pv: {
                 block_name_suffixed: {
```

Comment:
It occurs to me that it would be quite easy to make this backwards compatible, so we should do that to give people time to move over. Something like:
```python
def qsrv_field_access(pvi_field: str, panda_field: str, access: str) -> dict:
    block_name_suffixed = f"{pvi_field}.{panda_field.lower().replace(':', '_')}.{access}"
    return {
                block_name_suffixed: {
                    "+channel": "NAME",
                    "+type": "plain",
                    "+trigger": block_name_suffixed,
                }
            }
```
Then use it like:
```python
 record.add_info(
        "Q:group",
        {
            RecordName(f"{block}:PVI"): {
            pvi_pv: {
                ** qsrv_field_access("val", field, access),
                # Also add into `pvi` for back compat
                ** qsrv_field_access("pvi", field, access),
                }
            }
        },
    )
```

Reply:
That sounds good, I could also give a deprecation warning? I feel if I don't then users will probably still be unaware the change is coming and have the same scramble to change once we do actually get rid of the old structure.

Reply:
That would be nice, but I can't see how we would emit a deprecation warning, as we are just publishing a structure via QSRV, we have no idea what someone on the other end might actually be reading from that structure and QSRV gives us no hooks to know what the pvRequest was...

Reply:
I was thinking we just put a
``` python
warnings.warn("blah blah use .value", DeprecationWarning)
```
right before we start the interactive terminal.